newsroom/ui_config_async.py

```diff
@@ -0,0 +1,40 @@
+from typing import Optional, Dict, Any
+from superdesk.core.module import Module
+from superdesk.core.mongo import MongoResourceConfig
+from superdesk.core.resources import ResourceModel, ResourceModelConfig
+from superdesk.core.resources.service import AsyncResourceService
+from content_api import MONGO_PREFIX
+
+
+class UiConfig(ResourceModel):
+    preview: Optional[Dict[str, Any]] = None
+    details: Optional[Dict[str, Any]] = None
+    list: Optional[Dict[str, Any]] = None
+    advanced_search_tabs: Optional[Dict[str, Any]] = None
+    multi_select_topics: Optional[bool] = False
```

Comment:
If you're providing a default other than `None`, then the `Optional` annotation is not required.

Annotated with `Optional[<type>]` indicates that this variable accepts a value of `None`, which isn't always the case